include/glow/Base/Image.h

```diff
@@ -93,20 +93,51 @@ bool writePngImage(Tensor *T, const char *filename,
                    llvm::ArrayRef<float> mean = zeroMean,
                    llvm::ArrayRef<float> stddev = oneStd);
 
-/// Read a png image and preprocess it according to several parameters.
-/// \param filename the png file to read.
-/// \param imageNormMode normalize values to this range.
-/// \param imageChannelOrder the order of color channels.
+/// A helper function called by readPngImageAndPreprocess that does all the
+/// reading file and preprocessing work.
+/// \param imageData the tensor into which the preprocessed image data will be
+/// stored. \param filename the png file to read. \param imageNormMode normalize
+/// values to this range. \param imageChannelOrder the order of color channels.
 /// \param imageLayout the order of dimensions (channel, height, and width).
 /// \param mean use special mean to normalize.
 /// \param stdev use special stddev to normalize.
+void readPngImageAndPreprocessHelper(Tensor &imageData,
+                                     llvm::StringRef filename,
+                                     ImageNormalizationMode imageNormMode,
+                                     ImageChannelOrder imageChannelOrder,
+                                     ImageLayout imageLayout,
+                                     llvm::ArrayRef<float> mean = zeroMean,
+                                     llvm::ArrayRef<float> stddev = oneStd);
+
+/// Read a png image and preprocess it according to several parameters. Create a
+/// tensor and store the preprocessed image data into this tensor. \param
+/// filename the png file to read. \param imageNormMode normalize values to this
+/// range. \param imageChannelOrder the order of color channels. \param
+/// imageLayout the order of dimensions (channel, height, and width). \param
+/// mean use special mean to normalize. \param stdev use special stddev to
+/// normalize.
 Tensor readPngImageAndPreprocess(llvm::StringRef filename,
                                  ImageNormalizationMode imageNormMode,
                                  ImageChannelOrder imageChannelOrder,
                                  ImageLayout imageLayout,
                                  llvm::ArrayRef<float> mean = zeroMean,
                                  llvm::ArrayRef<float> stddev = oneStd);
 
+/// Read a png image and preprocess it according to several parameters. Take a
+/// tensor as a parameter and store the preprocessed image data into this
+/// tensor. \param imageData the tensor into which the preprocessed image data
```

Comment:
Bad formatting of the comment. Each `\param` should be at the beginning of a line.